tests/test_generate_success.py

```diff
@@ -0,0 +1,57 @@
+# Global imports
+import pytest
+from gatorgrade.generate.generate import generate_config
+
+"""Test for best case scenario"""
+
+
+@pytest.fixture()
+def setup(tmp_path):
+    """Setup directory for tests"""
+    # Given this file structure
+    src_dir = tmp_path / "src"
+    src_dir.mkdir()
+    python_file = src_dir / "test.py"
+    python_file.write_text("import sys")
+    writing_dir = tmp_path / "writing"
+    writing_dir.mkdir()
+    reflection = writing_dir / "reflection.md"
+    reflection.write_text("# Reflection")
+    yield tmp_path
+
+
+def test_generate_config_creates_gatorgrade_yml(setup):
+    """Test to see that gatorgrade_yml file exist in file structure"""
+    # When generate_config is called
+    generate_config(["src"], setup)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = setup / "gatorgrade.yml"
+    assert gatorgrade_yml.is_file()
+
+
+def test_generate_config_creates_gatorgrade_yml_with_dir_in_user_input(setup):
+    """Test to see if input matches directory"""
+    # When generate_config is called
+    generate_config(["src"], setup)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = setup / "gatorgrade.yml"
+    assert "src/test.py" in gatorgrade_yml.open().read()
+
```

Comment:
This function looks good and has no bugs to me. However, do you think it's a good idea that keep the assertion consistent with the function ` test_generate_config_creates_gatorgrade_yml_without_dir_not_in_user_input` below. If yes, you may check the existence of reflection here